spec/ruby/core/array/slice_spec.rb

```diff
@@ -165,6 +165,18 @@ def to.to_int() -2 end
       a.should == [1, 2]
     end
   end
+
+  ruby_version_is "2.7" do
+    it "works with beginless ranges" do
+      a = [0,1,2,3,4]
+      a.slice!(Range.new(nil, 3)).should == [0, 1, 2, 3]
```

Comment:
minor: could use dot notation for consistency (other places too)

Reply:
done! I kept on running into CI issues using dot notation, but I now realize I probably missed an eval somewhere.